Unity-Technologies/com.unity.multiplayer.samples.coop
# fix: simpler quitting flow

### Description (*)

This PR simplifies the condition in QuitPanel so that now it quits/sends to menu depending on the current active scene.
### Related Pull Requests

### Issue Number(s) (*)

Fixes issue(s): [MTT-2800](https://jira.unity3d.com/browse/MTT-2800)
### Manual testing scenarios

1. Try quitting from the main menu and from any other scene
2. See that in main menu the application quits while in other scenes you return to main menu
### Questions or comments

### Contribution checklist
